Components/ObjectViewController.swift

```diff
@@ -14,13 +14,21 @@ class ObjectViewController: UIViewController {
         return attributedString
     }()
 
+    let tableComponents: [TableRowModel] = {
+        let currency = "kroner"
+        return [
+            PriceTableComponent(title: "Prisantydning", price: 2500000, currency: currency),
```

Comment:
We should focus on the torget elements I don't think we have a price table component there 🤔

Reply:
That's true. I'll continue with torget specific stuff then.